Source/WebCore/platform/graphics/ImageBuffer.cpp

```diff
@@ -114,7 +114,7 @@ IntSize ImageBuffer::calculateBackendSize(FloatSize logicalSize, float resolutio
 
 ImageBufferBackendParameters ImageBuffer::backendParameters(const ImageBufferParameters& parameters)
 {
-    return { calculateBackendSize(parameters.logicalSize, parameters.resolutionScale), parameters.resolutionScale, parameters.colorSpace, parameters.pixelFormat, parameters.purpose };
+    return { calculateBackendSize(parameters.logicalSize, parameters.resolutionScale), parameters.resolutionScale, parameters.colorSpace, convertToPixelFormat(parameters.imageBufferPixelFormat), parameters.purpose };
```

Comment:
This is confusing. Why `ImageBufferParameters` has `ImageBufferPixelFormat` but `ImageBufferBackendParameters` has `PixelFormat`? Can't we have `ImageBufferPixelFormat` in both?